### ISSUE TITLE
ELB Integration failing when Cluster installed via Terraform

### ISSUE THREAD

**Issue by cricci82** [coreos/tectonic-installer#243]
Exposing a service of type='LoadBalancer' fails to provision the ELB in AWS.  

Controller Logs shows the following:
```
E0414 18:44:13.332619       1 servicecontroller.go:760] Failed to process service. Retrying in 5m0s: Failed to create load balancer for service policy-demo/hello-world: Multiple tagged security groups found for instance i-041e7788960d67752; ensure only the k8s security group is tagged
W0414 18:49:13.802150       1 aws.go:2751] Error opening ingress rules for the load balancer to the instances: Multiple tagged security groups found for instance i-041e7788960d67752; ensure only the k8s security group is tagged
E0414 18:49:13.802323     
```

**Comment by badbeeker** [comment]
It looks like this is a similar problem to https://github.com/kubernetes/kubernetes/issues/26787.  

The master and workers all have two security groups that are tagged KubernetesCluster=<value>.  Removing that specific tag from the **sg-cluster_default** security group that the tectonic installer sets up, fixes the issue and kubernetes was able to successfully create the ELB and assign the correct instances.

Perhaps the cluster default security group should not be tagged with KubernetesCluster=<value>??

**Comment by s-urbaniak** [comment]
#264 fixes this by assigning at most one security groups for workers/masters.

**Comment by s-urbaniak** [comment]
resolved by #264
